deepmd/utils/type_embed.py

```diff
@@ -127,6 +129,8 @@ def build(
         embedded_types
             The computational graph for embedded types
         """
+        if self.type_embedding_net_variables is not None:
+            return self.type_embedding_from_graph
```

Comment:
There will be three situations here when `self.type_embedding_net_variables is not None`:
- compress
- restart from an original model
- restart from a compressed model

The embedding network should be trainable for the second situation.